### PR TITLE
Improve the documentation around the use of ciphers

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1895.surge.sh

Updated pages:
- [Back up an online database](https://neo4j-docs-operations-1895.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Intra-cluster encryption](https://neo4j-docs-operations-1895.surge.sh/operations-manual/5/clustering/setup/encryption)
- [SSL framework](https://neo4j-docs-operations-1895.surge.sh/operations-manual/5/security/ssl-framework)

**Review by jennyowen — APPROVED** [review]
looks good. 
Also omg tabbed code blocks?! :exploding_head:

File: modules/ROOT/pages/clustering/setup/encryption.adoc [hunk 1]
```diff
@@ -104,41 +105,62 @@ By default, cluster communication is unencrypted.
...
 
 Add the following content to the _neo4j.conf_ file:
 
 [source, properties]
 ----
 dbms.ssl.policy.cluster.enabled=true
-dbms.ssl.policy.cluster.tls_versions=TLSv1.2
-dbms.ssl.policy.cluster.ciphers=TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
-dbms.ssl.policy.cluster.client_auth=REQUIRE
+dbms.ssl.policy.bolt.tls_versions=TLSv1.3,TLSv1.2 \ # <1>
```
Comment (renetapopova) [inline modules/ROOT/pages/clustering/setup/encryption.adoc:146]:
```suggestion
dbms.ssl.policy.cluster.tls_versions=TLSv1.3,TLSv1.2 \ # <1>
```

File: modules/ROOT/pages/clustering/setup/encryption.adoc [hunk 2]
```diff
@@ -104,41 +105,62 @@ By default, cluster communication is unencrypted.
...
 Add the following content to the _neo4j.conf_ file:
 
 [source, properties]
 ----
 dbms.ssl.policy.cluster.enabled=true
-dbms.ssl.policy.cluster.tls_versions=TLSv1.2
-dbms.ssl.policy.cluster.ciphers=TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
-dbms.ssl.policy.cluster.client_auth=REQUIRE
+dbms.ssl.policy.cluster.tls_versions=TLSv1.3,TLSv1.2 \ # <1>
+dbms.ssl.policy.bolt.ciphers=TLS_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_DHE_DSS_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA \ # <2>
```
Comment (renetapopova) [inline modules/ROOT/pages/clustering/setup/encryption.adoc:147]:
```suggestion
dbms.ssl.policy.cluster.ciphers=TLS_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_DHE_DSS_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA \ # <2>
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1895.surge.sh

Updated pages:
- [Back up an online database](https://neo4j-docs-operations-1895.surge.sh/operations-manual/5/backup-restore/online-backup)
- [Intra-cluster encryption](https://neo4j-docs-operations-1895.surge.sh/operations-manual/5/clustering/setup/encryption)
- [SSL framework](https://neo4j-docs-operations-1895.surge.sh/operations-manual/5/security/ssl-framework)

**Pull request merged**